csun-metalab/nomi
# NAM-320 -- UI: Update Gallery cards edit button

# Description
Re-positioned the edit button to the bottom right in gallery view.
  
# Testing
1. Click the grid view.
2. Ensure the edit button is in the proper position.
3. Ensure the edit button is responsive in different breakpoints.
